system/netdata.service.in

```diff
@@ -11,7 +11,7 @@ User=netdata
 Group=netdata
 RuntimeDirectory=netdata
 RuntimeDirectoryMode=0775
-PIDFile=@localstatedir_POST@/run/netdata/netdata.pid
+PIDFile=/run/netdata/netdata.pid
 ExecStart=@sbindir_POST@/netdata -P @localstatedir_POST@/run/netdata/netdata.pid -D
```

Comment:
Netdata creates a PID file set in ExecStart. Should we just remove `PIDFile`? @Ferroin

Reply:
https://www.freedesktop.org/software/systemd/man/systemd.service.html brief read: we need `PIDFile` set when `Type=forking`.

Reply:
Ah, yeah, this needs one minor change beyond what’s already here.